applications/openshift/controller/controller_rotate_kubelet_server_certs/rule.yml

```diff
@@ -63,7 +63,7 @@ warnings:
     of rotation yourself
 
 references:
-    cis@ocp4: 1.3.6
+    cis@ocp4: 4.2.10
```

Comment:
Correct reference seems to be `4.2.11`.
`4.2.10` is about `RotateKubeletClientCertificate`

Reply:
good catch, I will fix this